src/test/java/com/amazon/ion/system/IonTextWriterBuilderTestBase.java

```diff
@@ -124,71 +115,10 @@ public void testCatalogImmutability()
 
     //-------------------------------------------------------------------------
 
-    @Test
-    public void testInitialIvmHandling()
```

Comment:
Initial IVM is not optional for Ion 1.1, so these tests are not applicable for the 1.1 writer. They are moved into `IonTextWriterBuilder_1_0_Test`.